flocker/control/schema/types.yml

```diff
@@ -280,3 +280,37 @@ definitions:
       primary:
         '$ref': '#/definitions/primary'
     additionalProperties: false
+
+  lease_expiration:
+    title: "Lease Expiration"
+    description: |
+      The number of seconds until a lease expires, or null to indicate no expiration.
+    type:
+      - "number"
+      - "null"
+
+  lease_dataset_id:
+    title: "Dataset"
+    description: |
+      An opaque identifier, unique across the cluster, identifying a
+      particular dataset.
+    type: string
+    # The length of a stringified uuid
+    minLength: 36
+    maxLength: 36
+
+  lease:
+    type: object
+    description: "A specific lease."
```

Comment:
"A lease on a dataset" or something like that might be a better description, "a specific lease" doesn't really say anything more than you'd get from just the word "lease".